projects/laji/src/app/+trait-db/shared/trait-search/trait-search-filters/trait-search-filters.component.html

```diff
@@ -4,15 +4,15 @@
   <div class="mb-5 d-flex flex-gap-4">
     <div class="dataset-filter flex-grow">
       <label for="trait-search-filters-dataset">Dataset</label>
-      <select class="form-control" id="trait-search-filters-dataset" [formControl]="form.get('dataset')">
+      <select class="form-control" id="trait-search-filters-dataset" [formControl]="$any(form.get('dataset'))">
```

Comment:
What happens, if you don't add the $any? And can it be any other type than $any?